raphtory/src/db/task/window_eval_vertex.rs

```diff
@@ -17,15 +21,14 @@ pub struct WindowEvalVertex<'a, G: GraphViewOps, CS: ComputeState, S: 'static> {
     ss: usize,
     vertex: LocalVertexRef,
     pub(crate) graph: &'a G,
-    local_state: Option<&'a mut S>,
+    _local_state: Option<&'a mut S>,
```

Comment:
This not being used?

Reply:
@fabianmurariu

Reply:
I really hope that the entire window_eval_* will be gone